policy/modules/contrib/kdump.te

```diff
@@ -100,6 +103,9 @@ allow kdumpctl_t self:unix_stream_socket create_stream_socket_perms;
 manage_files_pattern(kdumpctl_t, kdump_lock_t, kdump_lock_t)
 files_lock_filetrans(kdumpctl_t, kdump_lock_t, file, "kdump")
 
+manage_files_pattern(kdumpctl_t, var_log_t, kdump_log_t)
```

Comment:
Correct. Updated it.

FYI, I thought it needs to be `var_log_t` since `kdumpctl_t` does need write permission on dirs with `var_log_t` type when `kdump.log` is created. But, I just realized that `filetrans_pattern($1, var_log_t, $2, $3, $4)` in `logging_log_filetrans()` does that job. So, we can have `kdump_log_t` in the second parameter of `manage_files_pattern()` as you pointed out.